nodejs/lesson04/solution/hello-context.js

```diff
@@ -80,6 +80,11 @@ function http_get(fn, url, span) {
                 span.finish();
                 return data;
             }, e => {
+                span.setTag(Tags.ERROR, true)
+                span.log({
+                    'event': 'error',
+                    'value': e.message
```

Comment:
s/value/error.object/

Reply:
I considered doing it, but error object would be too big here. It will log around 20 lines  What do you suggest?